single_tech_samples/fabric/feature_engineering_on_fabric/README.md

```diff
@@ -0,0 +1,365 @@
+
+After signing into Microsoft Fabric, it's necessary to create a new workspace for setting up a pipeline. Therefore, please navigate to the left-hand menu, select the workspace option, and proceed to create a new workspace.
+
+![workspace_01](./images/data_pipeline/workspace_01.png)
+
+Next, please enter your workspace name and description (optional). If you wish, you can upload a photo to use as your workspace avatar. This is also optional. However, please note that you need to select 'trial' under the 'license mode' in the advanced options, as currently only this type supports all the features of Fabric. Then click the 'Apply' button to complete the creation of the workspace.
+
+![workspace_02](./images/data_pipeline/workspace_02.jpg)
+
+##### Step 2: Create a Lakehouse
+
+Then within this created workspace, create a new Lakehouse named as '[prefix]_lh' by clicking the 'New' button on the top of the page. Please note that [prefix] is the prefix you defined in the prerequisites section.
+
+##### Step 3: Import the sample notebooks
+
+After the Lakehouse is created, please click the 'New' button again, and select 'Import notebook' from the menu. Navigate to `src/notebooks` folder under the repo, and select all the notebooks to import:
+
+![import all notebooks](./images/import_all_notebooks.png)
+
+##### Step 4: Add the Created Lakehouse to the notebooks
+
+After the notebooks are imported, please open the notebook named as 'data_cleansing'，'data_ingestion'，'data_transformation','data_validation'，‘EDA','feature_set_registration' respectively, and add the Lakehouse created in the previous step to the notebook. Please note that the Lakehouse name should be '[prefix]_lh', where [prefix] is the prefix you defined in the prerequisites section.
+
+![add lakehouse](./images/add_lh_to_notebook.png)
+
+#### Create an Azure Resource Group
+
+Create a new resource group named as [prefix]rg, where [prefix] is the prefix you defined in the prerequisites section.
+
+#### Azure App Service
+
+Create a new Azure App Service named as [prefix]as, where [prefix] is the prefix you defined in the prerequisites section.
+
+#### Azure Data Lake Storage Gen2
+
+Create an Azure Data Lake Storage Gen2 account named as [prefix]sa, where [prefix] is the prefix you defined in the prerequisites section. For the detailed information, please refer to: [Create a storage account to use with Azure Data Lake Storage Gen2](https://learn.microsoft.com/en-us/azure/storage/blobs/create-data-lake-storage-account)
+
+#### Azure Purview
+
+Create an Azure Purview account named as [prefix]pv, where [prefix] is the prefix you defined in the prerequisites section.
+
+#### Azure Machine Learning Managed Feature store
+
+Create an Azure Machine Learning Managed Feature store by running the following Azure CLI:
+
+```az ml feature-store create --subscription <subscription id> --resource-group <resource group> --location <location> --name <location>```
```

Comment:
Is there a specific reason why we are creating this using CLI? For this, we need to mention that the "ml" extension needs to be added first.
```
az extension add --name ml
```